SA-6 'Lupara' barrel modification

Unisol opened the pull request:
<!-- If you are new to the Harmony repository, please read the [Contributing Guidelines](https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md) -->
## About the PR
This PR adds a SA-6 'Lupara', barrel modification for double-barrel shotgun. When applied to standard double-barrel shotgun it adds four more barrels for a total of six barrels. Also works with sawn-off version of the double-barrel shotgun.

Textures provided by thesaladmancer.

## Why / Balance
This 8 TC purchase effectively turns double-barrel into kammerer.

This is a job-restricted item, available only to bartenders, zookeepers and service workers.

Purchase does not come with ammo or anything else beyond the kit itself. 

Six-barrel shotgun (and conversion kit) is considered major (not syndicate) contraband to make it properly illegal for everyone with option to legalize it.

Six-barrel shotgun occupies 5x2 space in inventory, while sawn-off six-barrel occupies 2x2 space.

## Media
https://youtu.be/Oj3BkILDGzw

## Requirements
<!-- Confirm the following by placing an X in the square brackets.
Correct: [X]
Incorrect: [ ] [X ] [ X] -->
- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

:cl:
- add: added barrel modification kit in uplink catalog for bartender and zookeeper

DieselMohawk commented on Resources/Textures/_Harmony/Objects/Weapons/Guns/Shotguns/6b_sawn.rsi/meta.json:
No need to mention Harmony in the Harmony namespace.

DieselMohawk commented on Resources/Textures/_Harmony/Objects/Weapons/Guns/Shotguns/6b_sawn_icon.rsi/meta.json:
No need to mention Harmony in the Harmony namespace.

DieselMohawk commented on Resources/Textures/_Harmony/Objects/Weapons/Guns/Shotguns/6b_shotgun.rsi/meta.json:
No need to mention Harmony in the Harmony namespace.

DieselMohawk commented on Resources/Textures/_Harmony/Objects/Weapons/Guns/Shotguns/6b_shotgun_icon.rsi/meta.json:
No need to mention Harmony in the Harmony namespace.

Unisol commented:
Removed Harmony mentions in art attributions
Added Major Contraband tag to six-barrel shotgun, six-barrel sawn off and six-barrel modification kit
Added six round burst to both shotguns.
